### PR TITLE
docs: Japanese translation 20201221

### PR DESCRIPTION
**Pull request by wahapo**
## Context
Update Japanese translation about [this comparison](https://github.com/screwdriver-cd/guide/compare/v0.0.310...v0.0.327).

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: docs/ja/cluster-management/configure-api.md [hunk 1]
```diff
@@ -242,7 +278,12 @@ K8S_BUILD_TIMEOUT | 90 | クラスタ内の全てのビルドのデフォルト
...
 K8S_NODE_SELECTORS | `{}` | pod のスケジューリング用の k8s の node selector (フォーマット `{ label: 'value' }`) <https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#step-one-attach-label-to-the-node>
 K8S_PREFERRED_NODE_SELECTORS | `{}`|  pod のスケジューリング用の k8s の node selector (フォーマット `{ label: 'value' }`) <https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity-beta-feature>
-DOCKER_FEATURE_ENABLED | false | ビルドポッド内でDocker In Dockerを有効にするフラグ |
+K8S_POD_DNS_POLICY | ClusterFirst  | ビルドポッドのDNSポリシー <https://kubernetes.io/docs/concepts/services-networking/dns-pod-service/#pod-s-dns-policy>
+K8S_POD_IMAGE_PULL_POLICY | Always | ビルドポッドイメージのPullポリシー <https://kubernetes.io/docs/concepts/containers/images/#updating-images>
+K8S_POD_LABELS | `{ app: 'screwdriver', tier: 'builds', sdbuild: buildContainerName }` | クラスタ設定のための k8s ポッドラベル (例: { network-egress: 'restricted' } とすると、デフォルトでパブリックなインターネットへのアクセスをブロックしてビルドを実行します。)
+K8S_IMAGE_PULL_SECRET_NAME | ''    | K8sのmagePullSecrets名 (オプション) <https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/#create-a-pod-that-uses-your-secret>
```
Comment (yoshwata) [inline docs/ja/cluster-management/configure-api.md]: > magePullSecrets

image?

File: docs/ja/cluster-management/configure-api.md [hunk 2]
```diff
@@ -574,6 +615,28 @@ rateLimit:
     duration: 60000
 ```
 
+### Canaryルーティング
+
+ScrewdriverのKubernetesクラスタが[nginx Canary ingress](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#canary)を利用している場合、この環境変数を設定することでAPIサーバに一定期間だけCookieをセットさせ、後続のAPIリクエストが同じCnaryのAPIポッドに割り振られるようにします。
```
Comment (yoshwata) [inline docs/ja/cluster-management/configure-api.md]: > 後続のAPIリクエストが同じCnary

Canary

File: docs/ja/cluster-management/configure-ui.md [hunk 3]
```diff
@@ -77,3 +95,22 @@ $ docker run -d -p 8000:80 -e AVATAR_HOSTNAME="avatars*.githubusercontent.com bi
...
+デフォルト: https://slack.screwdriver.cd/
+
+例:
+```bash
+$ docker run -d -p 8000:80 -e SLACK_URL=https://slack.mydomain.com screwdrivercd/ui:stable
+```
+
+### Canaryルーティング
+
+ScrewdriverのKubernetesクラスタが[nginx Canary ingress](https://kubernetes.github.io/ingress-nginx/user-guide/nginx-configuration/annotations/#canary)を利用している場合、この環境変数を設定することでUIサーバに一定期間だけCookieをセットさせ、後続のAPIリクエストが同じCnaryのUIポッドに割り振られるようにします。
```
Comment (yoshwata) [inline docs/ja/cluster-management/configure-ui.md]: > APIリクエストが同じCnary

canary

File: docs/ja/cluster-management/configure-api.md (hunk 1 above)
Comment (wahapo) [reply docs/ja/cluster-management/configure-api.md]: 🙈

**Comment by wahapo** [comment]
I added v0.0.327
https://github.com/screwdriver-cd/guide/compare/v0.0.326...v0.0.327

**Review by yoshwata — APPROVED** [review]

**Review by ibu1224 — APPROVED** [review]

**Review by tkyi — APPROVED** [review]

**Pull request merged**